sdk/include/ndk/iotypes.h

```diff
@@ -334,13 +334,50 @@ typedef enum _FILE_INFORMATION_CLASS
     FileNormalizedNameInformation,
     FileNetworkPhysicalNameInformation,
 #endif
-#if (NTDDI_VERSION >= NTDDI_WIN7)
+#if (NTDDI_VERSION >= NTDDI_WIN7) || defined(__REACTOS__)
     FileIdGlobalTxDirectoryInformation,
     FileIsRemoteDeviceInformation,
-    FileUnusedInformation,
+    FileAttributeCacheInformation, // FileUnusedInformation since Windows 8.1
     FileNumaNodeInformation,
     FileStandardLinkInformation,
     FileRemoteProtocolInformation,
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN8) || defined(__REACTOS__)
+    FileRenameInformationBypassAccessCheck = 56 /*0x38*/,
```

Comment:
Not sure keeping the hexadecimal helps there.

Reply:
I just copied the output from SymbolTypeViewer and wanted to see what the feedback is. I can also remove the decimals for consistency, on the other hand they can be useful, especially when you don't have intellisense or similar. Any opinion on that?